devsetup/scripts/tripleo.sh

```diff
@@ -101,14 +96,13 @@ export OCTAVIA_ENABLED=${OCTAVIA_ENABLED}
 export TELEMETRY_ENABLED=${TELEMETRY_ENABLED:-true}
 
 set +x
-if [[ -f \$HOME/containers-prepare-parameters.yaml ]]; then
-    echo "Using existing containers-prepare-parameters.yaml - contents:"
-    cat \$HOME/containers-prepare-parameters.yaml
-else
+if [ ! -f \$HOME/containers-prepare-parameters.yaml ]; then
```

Comment:
fair enough. 
We may even just remove the conditional altogether, though keeping it will still allow someone to override if they needed to.

for the record/for other reviewers, this was added when we used to deploy tripleo wallaby in upstream jobs but wanted to override that for the downstream job (where we specified a containers-prep file like https://code.engineering.redhat.com/gerrit/c/openstack/tripleo-ci-internal-jobs/+/445396/28/playbooks/data_plane_adoption/containers-prepare-parameters.yaml

Reply:
if we are going to keep this conditional if then can we please re-add the else with the echo statement

```
else
    echo "Using existing containers-prepare-parameters.yaml - contents:"
    cat \$HOME/containers-prepare-parameters.yaml
```

This isn't being used in any jobs any more but if we are leaving the ability for this in local run/deployment the debug info will be useful if someone DOES want to use a custom containers prep file.